app/models/batch_context.rb

```diff
@@ -188,6 +189,22 @@ def verify_staging_location_path
     errors.add(:staging_location, 'not a sub directory of allowed parent directories.')
   end
 
+  def verify_processing_configuration
+    if only_group_by_filename_processing_configuration? && processing_configuration == 'default'
+      errors.add(:processing_configuration, "must be 'Group by filename' for #{content_structure.titleize} content type")
+    end
+
+    errors.add(:processing_configuration, "must be 'Default' for #{content_structure.titleize} content type") if only_default_processing_configuration? && processing_configuration != 'default'
+  end
+
+  def only_default_processing_configuration?
+    %w[document file geo media 3d webarchive_seed].include?(content_structure)
+  end
+
+  def only_group_by_filename_processing_configuration?
+    %w[simple_image simple_book maps].include?(content_structure) && using_file_manifest.blank?
+  end
```

Comment:
really we should have `simple_book_rtl` here too, but that is going away here: https://github.com/sul-dlss/pre-assembly/pull/1453